openreview/openreview.py

```diff
@@ -76,28 +81,32 @@ def __handle_response(self,response):
             else:
                 raise OpenReviewException(response.json())
 
-    def __login_user(self,username=None, password=None):
 
+    ## PUBLIC FUNCTIONS
+
+    def login_user(self,username=None, password=None):
+        '''
+        Logs in a registered user and returns authentication token
+        '''
         if username==None:
             try:
                 username = os.environ["OPENREVIEW_USERNAME"]
             except KeyError:
-                username = builtins.input("Please provide your OpenReview username (e.g. username@umass.edu): ")
+                pass
 
         if password==None:
             try:
                 password = os.environ["OPENREVIEW_PASSWORD"]
             except KeyError:
-                password = getpass.getpass("Please provide your OpenReview password: ")
-
-        self.user = {'id':username,'password':password}
+                pass
 
-        response = requests.post(self.login_url, json=self.user)
+        user = {'id':username,'password':password}
+        header = {'User-Agent': 'test-create-script'}
+        response = requests.post(self.login_url, headers=header, json=user)
```

Comment:
referencing our conversation about this from the standup earlier today: so, if the user and password are None, I guess the server returns a guest token?